code/modules/reagents/Chemistry-Goon-Toxins.dm

```diff
@@ -26,25 +26,27 @@ datum/reagent/histamine
 	reagent_state = LIQUID
 	color = "#CF3600" // rgb: 207, 54, 0
 	metabolization_rate = 0.2
+	overdose_threshold = 30
 
 datum/reagent/histamine/on_mob_life(var/mob/living/M as mob)
 	if(!M) M = holder.my_atom
-	if(volume >= 30)
-		M.adjustOxyLoss(pick(2,5)*REM)
-		M.adjustBruteLoss(pick(2,5)*REM)
-		M.adjustToxLoss(pick(2,5)*REM)
-	if(volume < 30)
-		switch(pick(1, 2, 3))
-			if(1)
-				M << "<span class='danger'>You are unable to look straight!</span>"
-				M.Dizzy(10)
-			if(2)
-				M.emote("cough")
-				var/obj/item/I = M.get_active_hand()
-				if(I)
-					M.drop_item()
-			if(3)
-				M.adjustBruteLoss(5*REM)
+	switch(pick(1, 2, 3))
+		if(1)
+			M << "<span class='danger'>You are unable to look straight!</span>"
+			M.Dizzy(10)
+		if(2)
+			M.emote("cough")
+			var/obj/item/I = M.get_active_hand()
+			if(I)
+				M.drop_item()
+		if(3)
+			M.adjustBruteLoss(5*REM)
```

Comment:
Histamine really isn't as horrible as people make it out to be on Goon, but it IS freaking annoying due to the coughing, incessant blurriness, and dropping of stuff in your hands; it does cause brute, but not often.